debian/control

```diff
@@ -31,31 +31,33 @@ Recommends: python-matplotlib,
 Suggests: ipython
 Provides: ${python:Provides}
 XB-Python-Version: ${python:Versions}
-Description: toolbox for analysis of MR diffusion imaging data
- Dipy is a toolbox for the analysis of diffusion magnetic resonance
- imaging data. It features:
-  - Reconstruction algorithms, e.g. GQI, DTI
-  - Tractography generation algorithms, e.g. EuDX
-  - Intelligent downsampling of tracks
-  - Ultra fast tractography clustering
-  - Resampling datasets with anisotropic voxels to isotropic
-  - Visualizing multiple brains simultaneously
-  - Finding track correspondence between different brains
-  - Warping tractographies into another space, e.g. MNI space
-  - Reading many different file formats, e.g. Trackvis or NIfTI
-  - Dealing with huge tractographies without memory restrictions
-  - Playing with datasets interactively without storing
 
+Description: A Pythonic library for the analysis of diffusion MRI datasets.
+
+DIPY is a free and open source software project for computational neuroanatomy. It focuses on diffusion magnetic resonance imaging (dMRI) analysis and tractography but also contains implementations of other computational imaging methods such as denoising and registration that are applicable to the greater medical imaging and image processing communities. Additionally, DIPY is an international project which brings together scientists across labs and countries to share their state-of-the-art code and expertise in the same codebase, accelerating scientific research in medical imaging.
+
+Here are some of the highlights:
```

Comment:
FWIW -- that awkward formatting was for a reason ;)  but ok -- thanks -- I will merge and adjust it back a bit. Cheers